src/elements/autocomplete-grid/autocomplete-grid/autocomplete-grid.visual.spec.ts

```diff
@@ -150,7 +151,7 @@ describe('sbb-autocomplete-grid', () => {
       for (const borderless of [false, true]) {
```

Comment:
Somehow I like it more if directly after the for loop there is a describe() setting the name there. With that not every it() needs e.g. to include negative=, borderless=, ...  what do you think? (would also be the case for the non grid variant)

Reply:
Fix done! i went for a maximum of 2 'states' in each 'it' block

Reply:
thank you :-)